app/admin.go

```diff
@@ -156,6 +156,13 @@ func (a *App) GetConfig() *model.Config {
 	return cfg
 }
 
+func (a *App) GetEnvironmentConfig() map[string]interface{} {
+	json := model.StringInterfaceToJson(a.envConfig)
```

Comment:
I don't understand the reason behind this StringInterfaceToJson + StringInterfaceFromJson

Reply:
After some investigation, looks like the unique reason to do that copy is to not alter the original value during the "Sanitize" call, because in the rest of the code is edited the GetConfig result. As long as envConfig is not edited at all, is safe to pass the original envConfig variable.

Reply:
Good point. I forgot about the Sanitize call. I'll remove the copy for now since I think we can trust that the server won't modify that